CHANGELOG.adoc

```diff
@@ -3,6 +3,10 @@ All wesentlichen Änderungen für dieses Projekt werden hier dokumentiert.
 
 Das Format basiert auf link:https://keepachangelog.com/en/1.0.0[Keep a Changelog].
 
+## [12.36.4] - 2022-03-02
+
+#254 Automatische Setup der Tabellen xtcasUsers und xtcasAuthorities.
```

Comment:
In Zukunft am besten mitbeschreiben, was das Ziel damit is:
Diese Tabellen werden benötigt, damit Benutzer-Logins und deren Security-Token-Zuweisung für Nutzer über im SQL-Server gespeichert und geladen werden können.